vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/StaffSearchModule.java

```diff
@@ -0,0 +1,39 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import java.util.List;
+import java.util.Map;
+
+public class StaffSearchModule {
+
+    private List<Module> modules;
+
+    private Map<String, String> firstImageOfFirstSlideForModule;
```

Comment:
bad variable name: `firstImageOfFirstSlideForModule` sounds like one object but it's a map